Blankmon/src/application/Player.java

```diff
@@ -51,4 +51,14 @@ public Image getSprite()
 		return mPartyAnature;
 	}
 
+	public void addAnatures(Anature toAdd)
+	{
+		if(toAdd == null)
+			throw new IllegalArgumentException("toAdd was Null.");
+		
+		else if(mPartyAnature.size() == 6)
+			throw new IllegalArgumentException("The Player's Anature Party already has 6.");
```

Comment:
Is this something we want to be throwing an exception for? For now, it is fine but in the future, a prompt to the user will probably be the best course of action.

Which now that I think of it we will stumble upon during testing. So this works!

Reply:
Yeah, we can probably switch it out for something else later, I just put that there as a place holder.